headerfile.go

```diff
@@ -25,6 +25,18 @@ type HeaderFile struct {
 	lookupStruct      map[string]*Struct
 }
 
+func (h *HeaderFile) Structs() []*Struct {
```

Comment:
Please remove these getters, and just export them if you need them. This is not Go idiomatic afaik, and there is no additional security since one can simply edit the data(in some cases even the slice) from outside.